pkg/util/cmd/exit.go

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
I think this package should not be in `/pkg` and instead `/cmd`.

Reply:
I'm not sure what you mean; moving the package from `pkg/util/cmd/` to where?
The package is currently used for multiple commands (both controller and kubectl plugin); so it cannot be placed under 1 of these `cmd/...` dirs.

Reply:
+1 to this. Technically a breaking API change but we encourage people not to import cert-manager as a module anyway and the chances of them having used this specific package are surely slim.

Reply:
Ok